<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplinerDialog</class>
 <widget class="QDialog" name="SplinerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="gbObjects">
       <property name="title">
        <string>Objects</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Delete Item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Delete All</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Mapping method</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="mappersComboBox"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="settingsLayout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Cell dimensions</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineX">
          <property name="text">
           <string>25</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineY">
          <property name="text">
           <string>25</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>X</string>
          </property>
          <property name="buddy">
           <cstring>lineX</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Y</string>
          </property>
          <property name="buddy">
           <cstring>lineY</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Bounds</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineMinX">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineMaxX">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblMinX">
          <property name="text">
           <string>Min X</string>
          </property>
          <property name="buddy">
           <cstring>lineMinX</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lblMaxX">
          <property name="text">
           <string>Max X</string>
          </property>
          <property name="buddy">
           <cstring>lineMaxX</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblMinY">
          <property name="text">
           <string>Min Y</string>
          </property>
          <property name="buddy">
           <cstring>lineMinY</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lblMaxY">
          <property name="text">
           <string>Max Y</string>
          </property>
          <property name="buddy">
           <cstring>lineMaxY</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="lineMinY">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineMaxY">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="text">
           <string>NewSurface</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnCalculate">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mappersComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>SplinerDialog</receiver>
   <slot>setupMapper(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>SplinerDialog</receiver>
   <slot>onClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCalculate</sender>
   <signal>clicked()</signal>
   <receiver>SplinerDialog</receiver>
   <slot>onCalculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setupMapper(int)</slot>
  <slot>onClose()</slot>
  <slot>onCalculate()</slot>
 </slots>
</ui>
